pinc/LPage.inc

```diff
@@ -253,6 +253,20 @@ function project_continuity_test($project, $orig_state, $no_more_pages)
     }
 }
 
+class RoundInfo
+{
+    public $round_id;
+    public $username;
+    public $user_id;
```

Comment:
I've added a comment about RoundInfo.
The comment after the include is not appropriate now and we can remove the include.
But should we have the include statement in Lpage.inc? LPage works without it because `forum_interface.inc` is included via at least one other path (through Project.inc). What do @cpeel  and @bpfoley  think about this? Use the minimum necessary includes, or show an include for where every function used in the file is defined, or don't worry about it.

Reply:
In general we should not rely on transient includes and if a function is used in a file we should have an include that pulls in the file with that function (exceptions are `misc.inc` and some of the site structure files which the github action will alert if those are included so there's some checks against that).

Our includes are getting better but it's still a bit of a mess.

Reply:
> I've added a comment about RoundInfo.

My apologies for not being specific enough. When I saw `user_id`, I immediately thought of `users.u_id`, not the phpBB user_id. At least not until I saw the call to `get_forum_user_id()`. However, I'm fine with whatever others think works best. It's obvious once you read the rest of the code.

Reply:
That's a really good point Sharon. It would be clearer if we called the variable `forum_user_id`.